zipline/pipeline/loaders/events.py

```diff
@@ -148,7 +152,8 @@ def __init__(self,
                 raise ValueError(
                     WRONG_MANY_COL_DATA_FORMAT_ERROR.format(sid=k)
                 )
-
+        self.events_by_sid = {sid: df.dropna(subset=[self.event_date_col]) for
```

Comment:
@mtydykov can you confer with @llllllllll about how you were doing the filtering here?  This case seems like it should be bread and butter for blaze...